### PR TITLE
Change the appointment update request DTO to ensure time and duration are always sent

### PR DESCRIPTION
**Pull request by tomsmyers**
## What does this pull request do?

Change the appointment update request DTO to ensure time and duration are always sent

## What is the intent behind these changes?

fit new appointment model changes in the API.

### CODE REVIEW

File: server/routes/serviceProviderReferrals/editSessionForm.ts [hunk 1]
```diff
@@ -24,8 +24,8 @@ export default class EditSessionForm {
     return {
       error: null,
       paramsForUpdate: {
-        appointmentTime: dateResult.value.toISOString() ?? null,
-        durationInMinutes: durationResult.value.minutes ?? null,
+        appointmentTime: dateResult.value.toISOString(),
+        durationInMinutes: durationResult.value.minutes!,
```
Comment (tomsmyers) [inline server/routes/serviceProviderReferrals/editSessionForm.ts:28]: is this ok to do? does the validation step check that this will not be null?

Comment (Gweaton) [reply server/routes/serviceProviderReferrals/editSessionForm.ts:28]: The UI prevents form submission if any of those fields are empty, so it _should_ be safe

Comment (sldblog) [reply server/routes/serviceProviderReferrals/editSessionForm.ts:28]: there's a test with `'duration-minutes': '',` input and it seems to pass 🤷‍♂️

**Review by sldblog — APPROVED** [review]
LGTM from my perspective

**Pull request merged**